pennylane/qchem/openfermion_obs.py

```diff
@@ -105,6 +103,14 @@ def observable(fermion_ops, init_term=0, mapping="jordan_wigner", wires=None):
     + (0.075) [Z2]
     + (-0.075) [Z0 Z2]
     """
+    try:
+        # pylint: disable=import-outside-toplevel, unused-import
+        import openfermion, openfermionpyscf
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires openfermion, openfermionpyscf. They can be"
+            " installed with `pip install openfermion openfermionpyscf`."
```

Comment:
```suggestion
            " installed with pip install openfermion openfermionpyscf."
```
No need to include this since it shows in the terminal :)